packages/www/src/pages/index.tsx

```diff
@@ -186,9 +183,9 @@ export default function Homepage() {
           <div class="info sticky">
             <h2>🌐 Full i18n support</h2>
             <p>
-              Brisa has a built-in internationalization (i18n) support that
-              allows you to <b>translate</b> your pages and <b>routing</b>,
-              carrying only the translations you consume.
+              Brisa has built-in internationalization support that allows you to{" "}
+              <b>translate</b> your pages and <b>routes</b>, while downloading
```

Comment:
```suggestion
              <b>translate</b> your pages and <b>routes</b>, while loading
```

Reply:
I had my debate regarding this one, will loading convey the meaning we want here? This is why I went with downloading, but understand the difference is negligible. Perhaps @aralroca can help us here.